build.dist.xml

```diff
@@ -191,6 +191,8 @@
         <phingcall target="include-local-settings" />
         <!-- Copy settings file containing development values, but preserve any existing local settings. -->
         <copy file="${website.settings.local.php.example}" tofile="${website.settings.local.php}" overwrite="false" />
+        <!-- Let the site config decide the verbosity or error messages -->
+        <replaceregexp file="${website.settings.local.php}" pattern="(\$config\['system.logging'\]\['error_level'\].*)" replace="// $1" />
```

Comment:
To avoid re-enabling verbose logging on every site install, we will run this line only once when the file is created.